assets/src/edit-story/components/rangeInput/index.js

```diff
@@ -47,7 +52,7 @@ const Input = styled.input.attrs({
   min-width: 100px;
   cursor: pointer;
   outline: none;
-  background: #fff4;
+  background: #2f3131;
```

Comment:
same here, is there a reason for not using `theme`?

Reply:
Fixed in bfc43f74a. It's the ["Gray 8" foreground color](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Book?node-id=1286%3A457075), but it was missing in the theme, now added.